src/memory/qdrant.rs

```diff
@@ -170,9 +147,9 @@ impl MemorySystem for QdrantMemorySystem {
         let relevant_memories: Vec<RelevantMemory> = search_result
             .iter()
             .map(|point| {
-                let content: String = get_value(&point.payload, "content");
-                let recall: f32 = get_value(&point.payload, "recall");
-                let recency: f32 = get_value(&point.payload, "recency");
+                let json_string = serde_json::to_string(&point.payload).unwrap_or("".to_string());
```

Comment:
I would use `to_value` and `from_value` instead of `to_string` and `from_string`, it's less expensive and a little more clear as to what we're doing.